### ISSUE TITLE
Quickly show/hide all cards in hand

### ISSUE THREAD

**Issue by UltimateGeek** [ArnoldSmith86/virtualtabletop#21]
```gherkin
Feature: Show & hide hand
  As a game player
  I want to quickly show the cards in my hand
  So that I can follow game rules without having to drag cards one at a time to the board

Scenario Outline: Reveal hand
  Given a hand with <n> cards
  When I show my hand
  Then it shows all <n> cards to all the other players (indefinitely)
Examples:
  | n  |
  | 15 |
  | 2  |
  | 0  |

Scenario Outline: Hide hand
  Given a hand with <n> cards
  And I show my hand
  When I hide my hand
  Then all <n> of my cards are no longer visible to the other players
Examples:
  | n  |
  | 15 |
  | 2  |
  | 0  |
```

**Comment by robartsd** [comment]
Where would the hand be shown, all players hands are in the same position on their screens. Better solution would be for the game designer to provide a button that moves the cards to a holder that is visible to everyone and a button to put the cards back into the hand. Might be nice to provide game designers with a feature that allows items in a holder to only be interacted with by certain player(s).

**Comment by bjalder26** [comment]
To piggyback on robartsd, the button could also change the opacity of the holder, as well as possibly resize and move it, so it could be small, off to the side, and not visible when not needed, and normal size, in the middle of the screen, and fully opaque when needed.

**Comment by bjalder26** [comment]
Is there a specific application for this, because I think I could make a toggle button that could accomplish this.  Also, with PR #373 a moveXY button could move cards out of a holder, tag them, then move them back in on the next click.  I'm going to link this to that PR and if someone would like me to make a button that does exactly that, then I would be happy to do so.  I already have buttons that do all that on ther server for that PR, but it just isn't one button that toggles.

**Comment by bjalder26** [comment]
I made some buttons that toggle between showing a hand in a holder vs putting it in a hand, and showing a hand on the board vs putting it in a hand.  They utilize PR #373.  Is it okay to link this issue to that PR?

http://212.47.248.129:3373/showcards
It will be at this link after PR 373 is merged: https://virtualtabletop.io/PR373-showcards

**Comment by bjalder26** [comment]
I think PR 373 provides good solutions to this issue (shown in room linked below), so I went ahead and linked it.  If anyone feels otherwise, then feel free to unlink them, but then further explanation of this issue would be helpful. 

> I made some buttons that toggle between showing a hand in a holder vs putting it in a hand, and showing a hand on the board vs putting it in a hand. They utilize PR #373. Is it okay to link this issue to that PR?
> 
> http://212.47.248.129:3373/showcards
> It will be at this link after PR 373 is merged: https://virtualtabletop.io/PR373-showcards

**Comment by 96LawDawg** [comment]
Closed by #43.
